hooks/playbooks/control_plane_horizon.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Kustomize ControlPlane for horizon service
+  hosts: "{{ cifmw_target_hook_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: Ensure the kustomizations dir exists
+      ansible.builtin.file:
+        path: "{{ cifmw_basedir }}/artifacts/manifests/kustomizations/controlplane"
+        state: directory
+        mode: "0755"
+
+    - name: Create kustomize yaml to enable Horizon
+      ansible.builtin.copy:
+        dest: "{{ cifmw_basedir }}/artifacts/manifests/kustomizations/controlplane/66-horizon-kustomization.yaml"
```

Comment:
Why is the hook named 80 but the file is 60? I'd name it the same for consistency.

Reply:
Done , thanks